### DISCUSSION TITLE
model with bones exporting problem, need help plz

### DISCUSSION THREAD

**Discussion by 098-Zither** [ross-g/io_pdx_mesh#83] [Q&A] [answered]
Hello , dear Mr. Ross
Im a student from China, sorry that my english is not very good :(
Im having a problem exporting my blender (ver 3.6) model with bones ,the error describes is in the following

> Mesh export failed!

> Mesh head_back#M has vertices skinned to a group {VERTEX_KEYS} targeting a missing or excluded armature bone!Check all bones using the 'pdxignorejoint' property

> Python: Traceback (most recent call last):
File "C:\Program Files (x86)\Steam\steamapps\commonBlender\3.6\scipts\addons\io_pdx_mesh\pdx_blender_impot_expor.py"line 396, in get_mesh_skin_info 
bone_index = bone_name.index{group_names[group_index]}

ValueError:"VERTEX_KEYS"is not in list

The model was imported from a game and I know less about bones in Blender, could you please help me check the problem? very thanks  QAQ

**Comment by 098-Zither** [comment]
![image](https://github.com/ross-g/io_pdx_mesh/assets/140939833/46fca3d3-62b2-48d8-97cd-ee2e7bc6dfa9)
![image](https://github.com/ross-g/io_pdx_mesh/assets/140939833/ea973523-6534-4f47-bb12-0e95bdaf765b)

I've found the group Called "VERTEX_KEYS" but i dont know how to operate next

**Comment by ross-g** [comment, marked as answer]
If you have not created this yourself I am not sure I can help, the error implies that the model is missing required bones from the scene. If you don't want it to animate then you can just delete the armature modifier.

> The model was imported from a game

It's usually not allowed to rip models from one game for use in another. This looks like the Giant Death Robot from Civilization 6?

**Reply by 098-Zither** [reply]
Hello, dear MR.Ross
I'm very happy to tell you I've sloved this problem by deleting unuseful excess bones.

Thanks for your patient reply and your excellent plug-in for blender!

Wish you have a nice day!  QwQ



> It's usually not allowed to rip models from one game for use in another.

I'm so sorry to hear that, Sorry my understanding of copyright is not deep enough.  QAQ
